x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/parser/IdentifierBuilder.java

```diff
@@ -14,7 +16,17 @@ public String visitIdentifier(IdentifierContext ctx) {
         return ctx == null ? null : unquoteIdentifier(ctx.getText());
     }
 
+    @Override
+    public String visitQualifiedName(QualifiedNameContext ctx) {
+        if (ctx == null) {
+            return null;
+        }
+
+        // this is fine, because we've already checked for array indexes [...]
+        return Strings.collectionToDelimitedString(visitList(ctx.identifier(), String.class), ".");
+    }
+
     private static String unquoteIdentifier(String identifier) {
-        return identifier.replace("\"\"", "\"");
+        return identifier.replace("`", "");
```

Comment:
Oh, I just double checked, and it doesn't look like there's a version of the method for this since `''` is an empty character literal.
![image](https://user-images.githubusercontent.com/31489089/73380092-aa61ba80-4280-11ea-8add-c6e53b15f8a1.png)

Reply:
indeed - strings it is..